os/board/rtl8730e/src/component/soc/amebad2/fwlib/ram_common/ameba_i2c.c

```diff
@@ -631,6 +633,14 @@ u32 I2C_MasterWriteBrk(I2C_TypeDef *I2Cx, u8 *pBuf, u32 len)
 				I2C_ClearAllINT(I2Cx);
 				return cnt;
 			}
+
+			DelayUs(1);
```

Comment:
DelayUS is necessary?

Reply:
`if not required`
I think you should confirm whether this is required or not.
Anyway delay causes performance degradation even it's small. If possible, I think removal is better.